corehq/ex-submodules/pillowtop/feed/couch.py

```diff
@@ -1,22 +1,23 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 from couchdbkit import ChangesStream
+
 from pillowtop.dao.couch import CouchDocumentStore
 from pillowtop.feed.interface import ChangeFeed, Change
 from pillowtop.utils import force_seq_int
 
 
 class CouchChangeFeed(ChangeFeed):
 
-    def __init__(self, couch_db, include_docs, couch_filter=None, extra_couch_view_params=None):
```

Comment:
removed `include_docs` since it was always set to true and we need it in order to determine which kafka queue to send the change do